docs/AA--Javascript-in-EE/javscript-coding-standards.md

```diff
@@ -0,0 +1,32 @@
+## Javascript Coding Style/Standards
+
+For the most part, Event Espresso is following the WordPress coding standards for javascript in the new build process. There are a few exceptions, notably we do not follow the yoda conditionals rule.
+
+To assist with implementing and following our standards, our build process makes use of [eslint](https://eslint.org/) and the [WordPress eslint configuration](https://github.com/WordPress-Coding-Standards/eslint-plugin-wordpress).
+
+## NPM scripts
+
+To assist with linting and fixing any coding style problems, we have two npm scripts that can be used:
+
+```bash
+npm run lint
+```
+
+and
+
+```bash
+npm run lint:fix
+```
+
+The former will go through all js files in the `assets` folder and root path and report on any eslint rule violations.  The latter will automatically fix any rule violations for things that can be automatically fixed.
+
+## Setting up in PHPStorm
```

Comment:
the official name is "PhpStorm"
see: https://www.jetbrains.com/phpstorm/specials/phpstorm/phpstorm.html?gclid=EAIaIQobChMIzYDs29ey2gIVlLXACh2IpwTrEAAYASAAEgIRgvD_BwE&gclsrc=aw.ds.ds&dclid=CPy6iN3XstoCFYpXfgod-6wDrw